### ISSUE TITLE
Maintainers block in Auspice config JSON

### ISSUE THREAD

**Issue by trvrb** [nextstrain/augur#400]
@jameshadfield @emmahodcroft ---

Like in #399, I've been working through upgrading to `export v2` across existing builds (https://github.com/nextstrain/augur/issues/334#issuecomment-552014597) and noticed an inconsistency in Auspice config JSON. 

Currently, elements with attributes like members of `geo_resolutions` are objects with specified attributes like `{ "key": "country", "title": "Country Custom Title" },`. This is the case for `colorings` and `geo_resolutions`, and also `filters` in the proposed change in #399.

This leaves `maintainers` as the odd one out with specifying attributes as an array. I really like the the command line flag `--maintainers` (#327, #368), but I would think it would be smoother to have the Auspice config follow:

```
  "maintainers": [
    { "name": "Jane Doe", "url": "www.janedoe.com" },
    { "name": "Ravi Kupra", "url": "www.ravikupra.co.uk" }
  ],
```

Note that this mirrors the resulting JSON export where this looks like:

```
"meta": {
  ... 
    "maintainers": [
      {
        "name": "Trevor Bedford",
        "url": "http://bedford.io/team/trevor-bedford/"
      }
    ],
```

Generally, I think good to match up naming / structure in Auspice config and the resulting exported JSON. (This is the case with almost all the existing fields)

**Comment by jameshadfield** [comment]
👍 It seems sensible to have the config file mirror the produced JSONs as much as possible.

This would require:
* changes to the config schema: https://github.com/nextstrain/augur/blob/v6/augur/data/schema-auspice-config-v2.json#L52
* changes to the export v2 code
* changes to all core builds which use a config file -- see https://github.com/nextstrain/augur/issues/334

**Comment by trvrb** [comment]
I've just updated basically all the core builds to use `export v2` (https://github.com/nextstrain/augur/issues/334#issuecomment-552014597). I'm happy to make these fixes to `maintainers` and `colorings` throughout.

**Comment by trvrb** [comment]
Resolved in #405
